frontend/src/App.js

```diff
@@ -1,21 +1,24 @@
-import { useState, useEffect } from "react"; //useState hook
+import { BrowserRouter as Router, Routes, Route } from "react-router-dom";
+import Checkout from "./pages/Checkout";
+import Orders from "./pages/Orders";
+import SignIn from "./pages/SignIn";
+import ErrorPage from "./pages/ErrorPage";
+
+import { useState, useEffect } from "react";
 import Axios from "axios";
-// import { SpinnerCircular } from "spinners-react";
 
-import "./App.css";
 import Navbar from "./components/Navbar";
 import Content from "./components/Content";
+import "./App.css";
 
-const baseURL = "http://localhost:3001"; //move to .env
-
+// after authentication remove Main from /
 function App() {
-  const [loading, setLoading] = useState(false);
+  const [loading, setLoading] = useState(false); //use this for something?
```

Comment:
Hehe this is a nice way of making the user understand that the server is doing business and you have a state that it is currently loading.

But you are setting and managing what shall happen to it

Reply:
I'm thinking about adding some animation but didn't come across a proper one yet.

Reply:
@upulieh you can just use a spinner gif or a spinner css style. Google that :D 

Be mindfull of choices that provide value versus choices that are for neatness :D

Reply:
Thanks. I'll try